### DISCUSSION TITLE
Not useful recommendations

### DISCUSSION THREAD

**Discussion by andreirat** [community/community#33915] [Other Feature Feedback, Questions, & Ideas]
For " For you" tab I would like to see recommendations based on my work ( technologies i'm interested in, languages and so on ) and less activity of other people.

**Comment by d4kw1n** [comment]
You can go to each organization, technology or user that you are interested in and then choose to follow them, observe those things in the "following" section. As for the "for you" part, this is a feature that is still in beta, so you may have to wait a little longer. Hope useful to you 🤗

**Comment by wllnrds** [comment]
I'm in @andreirat.
I really apreciate the ideia, but should be possible remove what I don't wanna see anymore. I see the same recomendations to follow of people I've been working on, but I don't wanna follow them, they are boring and use git just as work rules with all repos privates.

**Comment by Rainyan** [comment]
I wish there was a "not interested, don't ever show this to me again" option in the Explore feed; sometimes I find myself looking at a repo, not realising I've already looked at it in the past and decided not to engage with it for reasons X, and wasting my time until I realise this again.
